exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/ParquetGroupScan.java

```diff
@@ -864,6 +872,14 @@ public String getDigest() {
     return toString();
   }
 
+  public void setCacheFileRoot(String cacheFileRoot) {
+    this.cacheFileRoot = cacheFileRoot;
+  }
+
+  public void setBatchSize(long batchSize) {
+    this.recommendedBatchSize = batchSize;
```

Comment:
This will check against the current value of the option rather than the max value? I thought we want to ensure never to exceed the MAX value of PARQUET_READER_RECORD_BATCH_SIZE - did I misunderstand? I looked into the code and it seems like we don't have access to the max value - we would need to add new methods to expose the same OR maybe invoke the validate method?